CHANGELOG.md

```diff
@@ -124,6 +124,38 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `ch.qos.logback:logback-core` from 1.2.13 to 1.5.3 ([#12519](https://github.com/opensearch-project/OpenSearch/pull/12519))
 
 ### Changed
```

Comment:
The changelog seems to be messed up :(

Reply:
fixed this, seems like a merge from main related :/